gen/dcos-config.yaml

```diff
@@ -2605,6 +2605,20 @@ package:
         # insecure_skip_verify = true
         [inputs.zookeeper.tags]
           dcos-component-name = "ZooKeeper"
+      # Reads metrics from ETCs's prometheus client
+      [[inputs.prometheus]]
+        ## An array of urls to scrape metrics from.
+        urls = ["http://localhost:2379/metrics"]
```

Comment:
Can you add a test for it like this https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/test_metrics.py#L476

Reply:
Yup, that's what I am writing now 😄